src/client/api/container.c

```diff
@@ -448,10 +448,10 @@ int
 daos_cont_set_owner(daos_handle_t coh, d_string_t user, d_string_t group,
 		    daos_event_t *ev)
 {
-	daos_prop_t	*prop;
-	uint32_t	nr = 0;
-	uint32_t	i = 0;
-	int		rc;
+	daos_prop_t *prop;
```

Comment:
whitespace only changes in this file?

Reply:
yea i originally had some change in here and clang-format did it's thing. then i moved my changes but those remain.